PGPAuth/src/main/java/org/lf_net/pgpunlocker/Server.java

```diff
@@ -59,8 +83,8 @@ public static Server deserialize(String serialized) {
             return new Server(parts[0], parts[1]);
         }
 
-        if (parts.length == 3) {
-            return new Server(parts[0], parts[1], parts[2]);
+        if (parts.length == 5) {
```

Comment:
do not change things in this function. It is only here to support config loading for configs generated by older versions of the app